server/src/main/kotlin/org/javacs/kt/KotlinLanguageServer.kt

```diff
@@ -100,18 +100,15 @@ class KotlinLanguageServer(
         val clientCapabilities = params.capabilities
         config.completion.snippets.enabled = clientCapabilities?.textDocument?.completion?.completionItem?.snippetSupport ?: false
 
-        if (clientCapabilities?.window?.workDoneProgress ?: false) {
+        if (clientCapabilities?.window?.workDoneProgress == true) {
```

Comment:
Admittedly this is a style choice, but I prefer using the elvis operator for nullable/optional bools since it clarifies that this isn't a typo (and will probably also generate a warning if it isn't nullable):

```suggestion
        if (clientCapabilities?.window?.workDoneProgress ?: false) {
```